app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb

```diff
@@ -36,6 +36,19 @@ def connect(options = {})
     end
   end
 
+  def api_endpoint_url(location)
+    api_endpoint_overrides = ::Settings.ems.ems_ibm_cloud_power_virtual_servers.api_endpoint_overrides
+
+    if api_endpoint_overrides.key?(location.to_sym)
+      url = api_endpoint_overrides[location.to_sym]
+    else
+      region = location.sub(/-\d$/, '')
+      url = "#{region}.power-iaas.cloud.ibm.com"
+    end
+
+    url
```

Comment:
If you don't need to do anything else with the `url` here you can just do:
```
def api_endpoint_url(location)
    api_endpoint_overrides = ::Settings.ems.ems_ibm_cloud_power_virtual_servers.api_endpoint_overrides
  if api_endpoint_overrides.key?(location.to_sym)
    api_endpoint_overrides[location.to_sym]
  else
    region = location.sub(/-\d$/, '')
    "#{region}.power-iaas.cloud.ibm.com"
  end
end
```

Reply:
Could even one-line it as `api_endpoint_overrides[location.to_sym] || "#{location.sub(/-\d$/, '')}.power-iaas.cloud.ibm.com"`